src/components/SlateEditor/plugins/definitionList/utils/toggleDefinitionList.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { Editor, Range, Element, Transforms } from 'slate';
+import { firstTextBlockElement } from '../../../utils/normalizationHelpers';
+import { TYPE_DEFINTION_DESCRIPTION, TYPE_DEFINTION_TERM, TYPE_DEFINTION_LIST } from '../types';
+import { isDefinitionListItem } from './isDefinitionListItem';
+import isOnlySelectionOfDefinitionList from './isOnlySelectionOfDefinitionList';
+
+export const toggleDefinitionList = (editor: Editor) => {
+  if (!Range.isRange(editor.selection)) {
+    return;
+  }
+  const isSelected = isOnlySelectionOfDefinitionList(editor);
+
+  if (isSelected) {
+    return Transforms.liftNodes(editor, {
+      match: (node, path) =>
+        Element.isElement(node) &&
+        (node.type === TYPE_DEFINTION_DESCRIPTION || node.type === TYPE_DEFINTION_TERM) &&
+        isDefinitionListItem(editor, path),
+      mode: 'all',
+    });
+  } else {
+    Editor.withoutNormalizing(editor, () => {
+      if (!Range.isRange(editor.selection)) {
+        return;
+      }
```

Comment:
Blir det ikke overkill å sjekke range to ganger?

Reply:
Hvis nei, kommenter hvrorfor.

Reply:
om du ikke sjekker dobbelt vil `editor.selection` være undefined i `Editor.withoutnormalizing` da dette er i sitt eget scope igjen.

Reply:
Isåfall, kommenter hvorfor!